### PR TITLE
Fix Templar meeting

### LINKED ISSUE #921: Templars do not have an "end meeting"
They are at the mercy of the timing of other meetings. It must be very stressful to have to type that fast haha

### PR DESCRIPTION
**Pull request by ghost**
Resolve #921

Templars do have a meeting, but previously did not have a prompt/input.

On EM, this was just to indicate "end meeting" as if it were a vote.

This is now added.

### CODE REVIEW

File: Games/types/Mafia/roles/cards/MeetWithTemplars.js [hunk 1]
```diff
@@ -7,8 +7,11 @@ module.exports = class MeetWithTemplar extends Card {
 
         this.meetings = {
             "Templar Meeting": {
+                actionName: "End Meeting",
                 states: ["Night"],
-                flags: ["group", "speech"]
+                flags: ["group", "speech", "voting", "mustAct", "noVeg"],
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/MeetWithTemplars.js:12]: mustAct was used so that inputType boolean would force the response to be Yes

**Review by R9H9 — COMMENTED** [review]
Would something like this work, @pikulet & @minokr?

**Comment by minokr** [comment]
I used the "custom" inputType because I was going for the EpicMafia style of having the "end meeting" as the dropdown option under "Templar Meeting". I realize the `actionName` property should be omitted, otherwise the option is "End Meeting -> End Meeting".

Ideally, I'd rather have "Meeting options -> End meeting" (dropdown select) over "End Meeting? -> Yes" (boolean) since it would be better in context to have a button to end it as a direct option rather than a boolean question where "No" would never be a valid answer since it would not proceed to the next state. I feel like it'd defeat the purpose of it being a boolean to begin with.

Since Templars will not have any other meeting options unlike a Jailor who may end the meeting with or without a kill, I can turn this into an "end meeting" button (non-boolean) instead of a dropdown if the current action system allows.

**Comment by minokr** [comment]
@R9H9 @pikulet thoughts on above?

**Comment by minokr** [comment]
Decided to keep it as a dropdown. It appears that buttons were only intended to be for VegKickMeeting and PregameReadyMeeting.

Furthermore, as a meeting, it does not show a list of players, nor an indication of whether the player has voted or not, aside from system messages.

Input type is now "select" instead of "custom", since it's still intended to be a dropdown select.
Action name is removed to have the meeting title be Templar Meeting.

**Review by Giga1396 — APPROVED** [review]

**Pull request merged**